demisto_sdk/commands/find_dependencies/find_dependencies.py

```diff
@@ -907,14 +946,21 @@ def build_dependency_graph(pack_id, id_set, verbose_file):
         return graph
 
     @staticmethod
-    def find_dependencies(pack_name, id_set_path='', debug_file_path=''):
+    def find_dependencies(pack_name, id_set_path='', debug_file_path='', echo_results=True,
```

Comment:
should be ordered by importance:
* pack_name
* id_set_path
* exclude_ignored_dependencies
* update_pack_metadata
* echo_results - switch to `silent_mode` / `quiet_mode`
* debug_file_path